r4/errors.bal

```diff
@@ -159,10 +169,10 @@ public isolated function createInternalFHIRError(string message, Severity errSer
 
 isolated function createTypedError(string message, Severity errServerity, IssueType code,
```

Comment:
let's introduce a method to the validator package containing similar content as this. That way we'll be able to keep all validator related utilities in one package without being needed to change r4 package.